lib/asciidoctor-bibtex/js/bibtex.rb

```diff
@@ -0,0 +1,35 @@
+module AsciidoctorBibtex
+  `const {Cite, plugins} = require('@citation-js/core')`
+  `require('@citation-js/plugin-bibtex')`
+  
+  module BibTeX
+    def self.open(path, options = {})
+      file = File.read(path, encoding: 'utf-8')
+      return Bibliography.new(`new Cite(#{file})`)
+    end
+
+    class Bibliography
+      def initialize(js_bibliography)
+          @js_bibliography = js_bibliography
+      end
+
+      def to_citeproc(options = {})
+        return @js_bibliography
```

Comment:
`to_citeproc` return the JS object directly?

Reply:
In Ruby the code for bibtex and citeproc was seperated. But in javascript both in done by the same lib. Therefore no conversion is needed. 

My goal was to not touch the original code if possible, therefore I implemented the original classes used even though they don't make too much sense here